src/odemis/driver/xt_client.py

```diff
@@ -358,6 +358,16 @@ def check_and_transfer_latest_package(self) -> None:
                     update.show()
                 else:
                     logging.warning("{} is a bad file in {} not transferring latest package.".format(ret, package.path))
+
+                # This version supports reading raw stage coordinates
+                if current_version is not None and bitness is not None:
+                    if current_version <= "1.12.0":
+                        raise ValueError("Required Delmic XT Adapter version shoulb be >= 1.12.0",
+                                         "How to update?\n\n1. Full stop Odemis and close Delmic XT Adapter.\n"
+                                         "2. Uninstall the old Delmic XT Adapter and install the correct version.")
+                else:
+                    logging.warning(
```

Comment:
Should this be an error ? How about if the Delmix XT adaptor is in development version?